App.tsx

```diff
@@ -30,6 +38,19 @@ export default function App() {
         }
     };
 
+    if (dog === true) {
```

Comment:
I assume you only have this in here for testing?
![image](https://github.com/kfoley123/FirstRunWiz/assets/6433506/7f361417-8788-4a42-b12b-8c06249412a1)